tests/UI/pages/BO/BObasePage.js

```diff
@@ -212,7 +212,7 @@ module.exports = class BOBasePage extends CommonPage {
    * @returns {Promise<void>}
    */
   async closeOnboardingModal(page) {
-    if (await this.elementVisible(page, this.onboardingCloseButton, 1000)) {
+    if (await this.elementVisible(page, this.onboardingCloseButton, 3000)) {
```

Comment:
That's not part of the scope of this PR, right ?

Reply:
```suggestion
    if (await this.elementVisible(page, this.onboardingCloseButton, 1000)) {
```

yes it can be deleted

Reply:
Done the timeout is reset to the original value !

Reply:
If that really solves the problem we were having in headfull, it deserves its own PR tho. Feel free to make another one :)